ignite/metrics/gan/fid.py

```diff
@@ -143,8 +143,10 @@ def __init__(
     @staticmethod
     def _online_update(features: torch.Tensor, total: torch.Tensor, sigma: torch.Tensor) -> None:
         total += features
-        for i, feat in enumerate(features):
-            sigma[i] += feat * features
+        if torch.__version__ <= "1.7.0":
```

Comment:
This does not work see how we are doing it in the codebase !

Reply:
You mean something like this?
https://github.com/pytorch/ignite/blob/692487ce01ec2d212ca1693ebe53a46f255f5584/tests/ignite/engine/test_create_supervised.py#L84

Reply:
yes